config/prod.exs

```diff
@@ -11,7 +11,7 @@ import Config
 # before starting your production server.
 config :fz_vpn,
   wg_path: "wg",
-  cli: FzVpn.CLI.Sandbox
+  wg_adapter: FzVpn.Interface.WGAdapter.Sandbox
```

Comment:
prod config seems redundant, as it's superseded by runtime configs. Maybe we can simply remove this?